src/main/java/de/ait/secondlife/controllers/CategoryController.java

```diff
@@ -71,32 +82,71 @@ public ResponseEntity<CategoryDto> add(@RequestBody @Valid NewCategoryDto dto){
+            @ApiResponse(responseCode = "403",
+                    description = "Forbidden",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "404",
+                    description = "Resource not found",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class)))
+    }
+    )
+    public ResponseEntity<CategoryDto> update(@PathVariable("category-id") Long categoryId, @RequestBody @Valid CategoryDto dto) {
+        return ResponseEntity.ok(service.update(categoryId, dto));
     }
 
     @Operation(summary = "Hiding category from the list of the categories", description = "Accessible only by admin, and only if the list of the offers related to this category is empty")
     @DeleteMapping("/{category-id}")
-    public ResponseEntity<CategoryDto> hideCategory(@PathVariable("category-id")Long categoryId){
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Category hided",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = CategoryDto.class))),
+            @ApiResponse(
+                    responseCode = "401",
+                    description = "Unauthorized",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "403",
+                    description = "Forbidden",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "404",
+                    description = "Resource not found",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class)))
+    }
+    )
+    public ResponseEntity<CategoryDto> hideCategory(@PathVariable("category-id") Long categoryId) {
         CategoryDto hiddenCategory = service.hide(categoryId);
         return ResponseEntity.ok(hiddenCategory);
     }
 
     @Operation(summary = "Activating category with id", description = "Accessible only by admin")
     @PatchMapping("/{category-id}/set-active")
-    @ApiResponse(
-            responseCode = "200",
-            description = "Category activated",
-            content = {@Content(mediaType = "application/json", schema = @Schema(implementation = CategoryDto.class))}
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Category activated",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = CategoryDto.class))),
+            @ApiResponse(
+                    responseCode = "401",
```

Comment:
you have removed this endpoint from SecurityConfig. now it's impossible to send patch requests.
but if we return the configuration, there will be same issue with 401 reproducing as in update() method

Reply:
returned endpoint